runtime/lua/vim/diagnostic.lua

```diff
@@ -753,7 +753,7 @@ end
 ---@field message string
 ---@field source nil|string
 ---@field code nil|string
----@field _tags { deprecated: boolean, unnecessary: boolean}
+---@field tags { deprecated: boolean, unnecessary: boolean}
```

Comment:
Can you keep this as `_tags` and adjust below.

Reply:
@lewis6991 just to be clear rather than renaming things to `tags`, I should change references to `tags` to be `_tags`?

Reply:
I've reversed the changes, hopefully that's what you meant